docs/business.yaml

```diff
@@ -106,6 +106,109 @@ paths:
                 type: object
                 properties: {}
               examples:
+                Incorporation-application-response:
+                  summary: Incorporation application response  
+                  value:
+                    errors:
+                      - null
+                    filing:
+                      business:
+                        identifier: TqcTmFjD9X  
+                      header:
+                        accountId: 3445
+                        affectedFilings: []
+                        availableOnPaperOnly: false
+                        certifiedBy: API Specs
+                        colinIds: []
+                        comments: []
+                        date: '2024-07-25T16:17:57.191108+00:00'
+                        deletionLocked: false
+                        effectiveDate: '2024-07-25T16:17:57.191129+00:00'
+                        email: no_one@never.get
+                        filingId: 150333
+                        inColinOnly: false
+                        isCorrected: false
+                        isCorrectionPending: false
+                        name: incorporationApplication
+                        status: DRAFT
+                        submitter: api-key-account-3445-sandbox
```

Comment:
Great work. I have a small suggestion regarding the submitter field. To prevent any potential misuse, perhaps we could use a more generic placeholder. 
This is how we handle this field: `submitter: mocked submitter`